### ISSUE TITLE
unexpected error (-1) when i scrobble

### ISSUE THREAD

**Issue by FlashlightET** [elamperti/OpenWebScrobbler#208]
**Describe the bug**
when i scrobble an error pops up but the scrobble goes through

**Steps to reproduce**
1. Go to openscrobbler
2. scrobble song
3. See error

**Expected behavior**
song is scrobbled without unexpected error (-1)

**Screenshots**
![image](https://github.com/elamperti/OpenWebScrobbler/assets/29938499/e2251cbf-1a7e-4a3e-ac36-53558eee6355)
![image](https://github.com/elamperti/OpenWebScrobbler/assets/29938499/c4cc96fb-50a5-4c62-a14d-0ec5aac48479)

**Device**
  * [ ✔️  ] Desktop
  * [ ] Mobile

**Platform (please complete the following information):**
 - Operating system:  Windows 11
 - Browser (name and version): Google Chrome Version 114.0.5735.110 (Official Build) (64-bit)
 - Version (see footer of website): v2.7.9

**Additional context**
the errors stack but can be closed
*it also shows up when clicking "your profile" so it could be related to that*

**Comment by elamperti** [comment]
Hi @FlashlightET, this happens sometimes when the last.fm authentication has been revoked. Have you tried singing out and starting over?

If that doesn't help, please try using [this reset link](https://openscrobbler.com/reset.html), it will discard your session and locally cached files (don't worry, your scrobbles are safe!) - This should fix the issue if it comes from the website itself. If the problem persists even after the reset, it would suggest an issue contacting either Open Scrobbler server or the last.fm API. In that case you can try using a different internet connection (e.g. using cellular, a public wi-fi network, a friend's network, etc.).

To summarize:

1. Try signing out and starting over
2. If that doesn't work, use the reset link
3. If it still breaks, try a different network

Hope this helps! Let me know if it doesn't.

**Comment by elamperti** [comment]
Given there was no answer, I'll assume you were able to solve this issue. Feel free to reopen the issue if you need more help.
